### DISCUSSION TITLE
Join in Prisma

### DISCUSSION THREAD

**Discussion by arthur-demenskiy** [prisma/prisma#17436] [Q&A]
Hello I need to make request like

SELECT * FROM "A" INNER JOIN "B" ON "A".id = "B".a_id;

I have tried a lot of options, but Prisma all the time generates LEFT JOIN relationship. Google and documentation have no information about such cases
I stuck on it for almost 4 days, will appreciate any suggestions

Also do prisma have any instructions how to use left, right, inner and ohert joins like typeORM ?

**Comment by janpio** [comment]
What data do you want to get with your query?
Prisma generally is not a query builder that you tell which part of SQL to use, but which data you want - and it then creates the query for it and runs it.

**Reply by nurul3101** [reply]
To add, if you would like us to provide functionality of a query builder, I would recommend adding a 👍 to this feature request.
- https://github.com/prisma/prisma/issues/13017

Having more demand from users would help our product team in prioritising it.

**Reply by lyudad** [reply]
I need to get same that inner join does: return data that exist in A and B tables, now anything I do I get: everything that is in table A + data from B that match with A
<img width="609" alt="Снимок экрана 2023-01-20 в 11 26 11" src="https://user-images.githubusercontent.com/25103090/213673298-ddd9ac11-0cc1-4611-b0c6-813305e7cfe4.png">

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
